events/exchange/exchange.go

```diff
@@ -145,6 +145,17 @@ func (e *Exchange) Subscribe(ctx context.Context, fs ...string) (ch <-chan *even
 	ch = evch
 	errs = errq
 
+	ns, ok := namespaces.Namespace(ctx)
```

Comment:
SGTM if there is no such case that the client, using `A` namespace, subscribes  `B` namespace event.

No idea why `event.Subscribe` doesn't require namespace right now. 
As far as I know, the moby project uses `namespace=moby` in the filter and 
CRI plugin checks event's namespace at first. If there is such case using `namespace=B` 
with ctx.include(A namespace), this change will bring unexpected result I think.
But it looks rare...

REF:
1. moby subcribes event: https://github.com/moby/moby/blob/v22.06.0-beta.0/libcontainerd/remote/client.go#L777
2. CRI plugin: https://github.com/containerd/containerd/blob/main/pkg/cri/server/events.go#L254

Reply:
interesting .. I was just talking about the need for event subscription by namespace for our kubernetes PLEG work..

Reply:
@mikebrow Is the namespace you mentioned kubernetes namespace?